### ISSUE TITLE
Environmental hazards problem

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#1110]
Suit Protection immediately depleted, It would be never restore.

---

## Details 
your gear's (suit, helmet, pack) all environmental protections are instantly depleted. Probably at the moment you land on a planet = even before you get out into the dangerous environment.

and stay that way while the system is bugged. They never recover as they should when you are in any safe areas = inside your ship, inside a base etc.

So basically while you are affected by the bug, you are never protected from anything. The gear stats mean nothing.

## Game Version
1.13.61

## Language
English (EN)

## New Game
No

## Reproduction Steps
it triggers randomly. For me sometimes everything works, another time, on the same planet, Suit Protection immediately depleted.

**Comment by Deebz96** [comment]
This seems to be intentional. Turning on the Advanced option for environmental damage will make the protection work more as you'd expect. Changing this would affect the vanilla balance of the game.

**Comment by Constellation-VASCO** [comment]
To be precise, the suit protection does not recover in the ship or in the airlock. This system works well inside the dungeon, but it does not work outdoors.
Well explained in this post. https://www.reddit.com/r/Starfield/comments/169f0wn/suit_protection_immediately_depleted/
Sorry English is not my first language.

**Comment by Deebz96** [comment]
This is true for planets which have ambient dangerous conditions, such as extremely cold/hot temperatures. With the standard environmental system, suit protection seems to be more intended to protect against weather and hazards, rather than ambient conditions.

When Bethesda implemented the advanced environmental system, they made suit protection work for ambient dangerous conditions, but left standard working the same as always. I believe this is intentional. I'll agree that it isn't great design, since it can occasionally lead to you getting frostbite the second you step outside your ship, but it would be out of the scope of this mod to adjust the game's balance there.

**Comment by Constellation-VASCO** [comment]
I'm sorry. please check this video. https://youtu.be/GdeEDO2WucA
This is really intentional?

**Comment by Deebz96** [comment]
I believe so, yes. Set the "Environmental Damage & Afflictions" setting to Advanced, and you'll see more of what I mean. I assume they made protection work more as you'd expect with that setting, because you will also take health damage in these environments with it enabled. Protection acts as a small buffer from that in this case.

The standard setting was left working as you showed in the video, and has been that way since launch. With that setting, it seems that suit protection is only meant to work on planets that are normally hospitable. It provides a temporary buffer from hazardous conditions you may find. On planets that are normally inhospitable, it seems that you are always meant to be in danger when walking on those.

I agree that this doesn't make much sense, but it would be out of the scope of the SFCP to make balance changes like this. That would be the role of another mod.

**Comment by Constellation-VASCO** [comment]
I just completely understood, thank you for the detailed explanation.
